src/payment/checkout/payment-form/CardHolderInformation.jsx

```diff
@@ -145,7 +145,6 @@ export class CardHolderInformationComponent extends React.Component {
                 component={FormSelect}
                 options={this.renderCountryOptions()}
                 required
-                onChange={this.handleSelectCountry}
```

Comment:
https://github.com/edx/frontend-app-payment/blob/11e13aa332eb974ac397f20a62add9bfcc325e15/src/payment/checkout/payment-form/CardHolderInformation.jsx#L21-L24 is only used by https://github.com/edx/frontend-app-payment/blob/11e13aa332eb974ac397f20a62add9bfcc325e15/src/payment/checkout/payment-form/CardHolderInformation.jsx#L336-L337
which is not a field in the experiment, so I'm removing the handler for the experiment just to make sure that's not connected to what I'm seeing